torntools/views/settings/settings.html

```diff
@@ -188,6 +188,9 @@ <h2>Preferences</h2>
 								<input type="checkbox" />Display Vault balance under "Money" in Torn's Information tab
 							</div>
 							<div class="option tabbed" id="global-vault_balance_own"><input type="checkbox" />Display your own vault share</div>
+							<div class="option multi-nr-input" id="global-upkeep_more_than">
+								Highlight Properties when greater than : <input type="number" />
```

Comment:
New and a notice that it doesn't work for spouses (and maybe renters, still figuring that out).

Reply:
Done.